packages/components/src/local/organisms/planning-channel/index.tsx

```diff
@@ -765,11 +765,15 @@ export const PlanningChannel: React.FC<PropTypes> = ({
   }
 
   const onSupportPanelLayoutChange = (key: string, value: string) => {
```

Comment:
hello @philb89 :-) 
I Iike the new strategy, but can we make the key "support.planning.[rolename]", then we just call the children "filterApplied", "supportPanelWidth" and "visibleColumns".

Reply:
sure Ian 👍🏻